pkg/crc/network/nameservers.go

```diff
@@ -32,15 +34,51 @@ func GetResolvValuesFromInstance(sshRunner *ssh.Runner) (*ResolvFileValues, erro
 }
 
 func CreateResolvFileOnInstance(sshRunner *ssh.Runner, resolvFileValues ResolvFileValues) error {
-	resolvFile, _ := CreateResolvFile(resolvFileValues)
+	sd := systemd.NewInstanceSystemdCommander(sshRunner)
+	// Check if ovs-configuration.service exist and if not then it is old bundle and use the same way to
+	// update resolve.conf file
+	if state, err := sd.Status("ovs-configuration.service"); err != nil || state == states.NotFound {
+		if err := replaceResolvConfFile(sshRunner, resolvFileValues); err != nil {
+			return fmt.Errorf("error updating resolve.conf file: %s", err)
+		}
+		return nil
+	}
+
+	if err := sd.Start("ovs-configuration.service"); err != nil {
+		return err
+	}
+
+	if err := updateNetworkManagerConfig(sshRunner, resolvFileValues); err != nil {
+		return fmt.Errorf("failed to update NetworkManager configuration: %s", err)
+	}
+
+	return sd.Restart("NetworkManager.service")
```

Comment:
Is this required? it's odd if settings done with `nmcli` are not applied immediatly? I'd call this from `updateNetworkManagerConfig`

Reply:
Yes it is required.

Reply:
updated it to use from `updateNetworkManagerConfig`